sim/shaman/chain_lightning.go

```diff
@@ -41,6 +41,7 @@ func (shaman *Shaman) newChainLightningSpell(isElementalOverload bool) *core.Spe
 		for hitIndex := int32(0); hitIndex < numHits; hitIndex++ {
 			totalDamage := baseDamage * bounceReduction
 
+			//TODO : damage reduction from glyph need to be applied to spell power part of the damage also
```

Comment:
To achieve this TODO instead of `baseDamage * bounceReduction` do `spell.DamageMultiplier *= bounceReduction` in the calc loop then do `spell.DamageMultiplier /= bounceReduction` in the deal loop